src/FacebookPixel/Assets.php

```diff
@@ -59,12 +58,21 @@ public static function loadFrontendAssets() {
 				true
 			);
 
-			$session	= new DonationAccessor();
-			$donation	= new Donation( $session->getDonationId() );
+			$purchase = Give()->session->get('give_purchase');
```

Comment:
@henryholtgeerts could you explain the reasoning for replacing the `DonationAccessor` by directly accessing the session off of the global?

Reply:
Reading further, is it because you need `$purchase` to check for `give_create_account`, which perhaps isn't available via `DonationAccessor`?

Reply:
@kjohnson Yep, that's exactly the reason 👍